s3/pkg/datastore/driver/clean.go

```diff
@@ -0,0 +1,17 @@
+package driver
+
+type Closer interface {
+	Close()
+}
+
+var closers []Closer
+
+func AddCloser(closer Closer) {
```

Comment:
please consider thread safety here

Reply:
Hi rhsakarpos, the AddCloser will only be called in init func of the module. So it is thread safe.